src/WebJobs.Script.WebHost/ContainerManagement/LinuxContainerInitializationHostService.cs

```diff
@@ -55,6 +55,11 @@ private async Task ApplyStartContextIfPresent()
 
                 bool success = _instanceManager.StartAssignment(assignmentContext);
                 _logger.LogInformation($"StartAssignment invoked (Success={success})");
+
+                var error = await _instanceManager.SpecializeMSISidecar(assignmentContext);
+                _logger.LogInformation(!string.IsNullOrEmpty(error)
+                    ? $"Re-specializing MSI sidecar failed with {error}"
```

Comment:
nit: if error can contain spaces, enclose in single quotes?

Reply:
Also, wondering if we need a new log statement here - SpecializeMSISidecar already logs its result internally right? Perhaps it's best to just use those logs.

Reply:
Yes. Removing this.